header-tips/index.md

```diff
@@ -0,0 +1,19 @@
+## Quick tips for making a header
+
+If you don't want to make a custom image for your header, use a screenshot. A lot of screenshots turn out really nicely when blown up to header size.
+
+When taking your screenshot, run the game full-screen or in the highest resolution it supports. Make sure to capture the image as a JPG if it is pixel art, and as a PNG otherwise.
```

Comment:
Wouldn't you want to capture it as PNG if it is pixel art? Doesn't JPG make pixel art look blurry? Can we ask David/Nik/Yariv?